lms/djangoapps/verify_student/views.py

```diff
@@ -885,14 +885,13 @@ def _compose_message_reverification_email(
         context = {
             "status": status,
             "course_name": course.display_name_with_default,
-            "assessment": reverification_block.related_assessment,
-            "courseware_url": redirect_url
+            "assessment": reverification_block.related_assessment
         }
 
         # Allowed attempts is 1 if not set on verification block
         allowed_attempts = 1 if reverification_block.attempts == 0 else reverification_block.attempts
```

Comment:
From Will.
Looking through the code, I'm confused by this as well. What I'm seeing in the reverification block code doesn't match what I remember we discussed with Griff.

The value that course teams enter in Studio means the number of additional re-verification attempts after the first one. So if it's 0, that means that means the user gets one re-verification attempt by default. If it's 1, then the user gets two attempts (the first plus one additional). And so on (2 --> 3, 3 --> 4).

Does that make sense?